src/Message/AbstractRequest.php

```diff
@@ -229,6 +229,40 @@ protected function hydrateCardHolderData(DOMDocument $xml)
 
     // endregion
     // region Parameters
+    
+    /**
+     * Get the card data.
+     *
+     * Because the heartland gateway uses a common format for passing
+     * card data to the API, this function can be called to get the
+     * data from the associated card object in the format that the
+     * API requires.
+     *
+     * @return array
+     */
+    protected function getCardData() {
```

Comment:
Hmm. I'm not seeing it. Could you link me to it? Also, if it's only used in the tests, is there any way to move this to a test helper or something similar?